scholia/app/templates/wikiproject_maintained.sparql

```diff
@@ -1,8 +1,9 @@
+{% import 'sparql-helpers.sparql' as sparql_helpers -%}
 PREFIX bd: <http://www.bigdata.com/rdf#>
 PREFIX wdt: <http://www.wikidata.org/prop/direct/>
 PREFIX wikibase: <http://wikiba.se/ontology#>
 PREFIX wd: <http://www.wikidata.org/entity/>
-SELECT ?type (REPLACE(STR(?type), ".*/", "") AS ?typeLabel) ?count WHERE {
+SELECT ?type ?typeLabel ?count WHERE {
   {
     SELECT DISTINCT ?type (COUNT(?item) AS ?count) WHERE {
       ?item wdt:P6104 bd:{{ q }};
```

Comment:
This is still broken.

Reply:
And what's the problem here?  The query appears to work, but returns no results.